Tests/PathManagerTests.cs

```diff
@@ -12,8 +13,12 @@ public class PathManagerTests
         [Fact]
         public void GetFilesInDirectoryCorrectlyReturnsListOfFiles()
         {
```

Comment:
I feel like this test is doing a bunch of work that is not really relevant to it, just because the code it wants to test is in `PathManager`. The only thing that `GetFilesInDirectory` depends on in `PathManager` is the file extensions. I wonder if we could refactor that out of `PathManager` and make this code easier to test?